Source/CarthageKit/BuildOptions.swift

```diff
@@ -15,19 +15,24 @@ public struct BuildOptions {
 	/// Whether to use downloaded binaries if possible.
 	public var useBinaries: Bool
 
+	/// Whether to use xcframeworks or not
+	public var useXCFrameworks: Bool
```

Comment:
Can we do this automatically when using Xcode 11+?

Reply:
Are you suggesting to default automatically to xcframeworks if Xcode >= 11 ?

We can but, for macOS carthage will still produce a .framework (since packing just 1 framework has no effect)